src/uniprotkb/components/entry/similar-proteins/SimilarProteinsTable.tsx

```diff
@@ -80,12 +92,22 @@ const SimilarProteinsTable: FC<{ members: string[] }> = ({ members }) => {
   }
 
   return (
-    <DataTable
-      data={data.results}
-      columns={columnConfig}
-      getIdKey={(row) => row.primaryAccession}
-      density="compact"
-    />
+    <>
+      <DataTable
+        data={data.results}
+        columns={columnConfig}
+        getIdKey={(row) => row.primaryAccession}
+        density="compact"
+      />
+      <Link
+        to={{
+          pathname: LocationToPath[Location.UniProtKBResults],
+          search: `query=${query}`,
+        }}
+      >
+        Show all {total} UniProtKB {pluralise('entry', total, 'entries')}
```

Comment:
```suggestion
        Show all ({total}) UniProtKB entries in this cluster
```

- Just trying to avoid the case of `Show all 1 UniProtKB entry` --> `Show all (1) UniProtKB entries in this cluster`
- Thought `in this cluster` might make it a little clearer

Reply:
It's actually not "in this cluster" as it's the UniProtKB entries in the cluster minus the one(s) that we are looking at now.
Hence, if I'm looking at "Similar proteins" to P05067 (canonical), I don't want to see P05067 itself. It's not "similar", it is itself.
But probably need UX input on that on expected behaviour, especially knowing that we have people that are not too keen on highlighting the "cluster" concept in this section

Reply:
Is our wording good now?

Reply:
I think it's fine, yes. If we get different feedback about this specifically this is something we can change easily anyway